addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -207,7 +244,6 @@ export class ImageCrop extends Component {
         this.initialSrc = await applyModifications(this.media, {forceModification: true, mimetype: this.mimetype});
         cropped = this.aspectRatio === "0/0" ? false : cropped;
         this.media.classList.toggle('o_we_image_cropped', cropped);
-        this.$media.trigger('image_cropped');
```

Comment:
Code suggestion => to call the trigger only if needed, instead of completely removing it 

Again, may be useless if you use the styles for the previews :wink: 

```js
/**
 * Updates the DOM image with cropped data and associates required
 * information for a potential future save (where required cropped data
 * attachments will be created).
 *
 * @private
 * @param {boolean} [cropped=true]
 * @param {boolean} [refreshOptions=true] tells if the `image_cropped` event
 *     should be triggered (to rerender options).
 */
async _save(cropped = true, refreshOptions = true) {
    // Mark the media for later creation of cropped attachment
    this.media.classList.add('o_modified_image_to_save');

    [...cropperDataFields, 'aspectRatio'].forEach(attr => {
        delete this.media.dataset[attr];
        const value = this._getAttributeValue(attr);
        if (value) {
            this.media.dataset[attr] = value;
        }
    });
    delete this.media.dataset.resizeWidth;
    this.initialSrc = await applyModifications(this.media, {forceModification: true, mimetype: this.mimetype});
    cropped = this.aspectRatio === "0/0" ? false : cropped;
    this.media.classList.toggle('o_we_image_cropped', cropped);
    if (refreshOptions) {
        this.$media.trigger('image_cropped');
    }
    this._closeCropper();
}
```